wfe-web/src/main/java/ru/runa/common/WebResources.java

```diff
@@ -201,4 +201,7 @@ public static boolean isChatEnabled() {
         return RESOURCES.getBooleanProperty("chat.enabled", true);
     }
 
+    public static String getCustomTemplateForProcessStartMessage() {
```

Comment:
```suggestion
    public static String getProcessStartedMessage() {
```
т.к. мя свойства - `process.started`